runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -748,6 +749,7 @@ protected void unmapSecondaryIndexes(K key, V value) {
      * @param key   the primary key associated with the indexing.
      * @param value the value to map.
      */
+    @DontInstrument
     @SuppressWarnings("unchecked")
     protected void mapSecondaryIndexes(K key, V value) {
```

Comment:
`CorfuTable::insert` used to sync on maoSecondaryIndexes, but the method is a pure mutator